src/utils/helpers/browser-type-check/browser-type-check.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Check for Chrome or Firefox.
+ */
+function browserTypeCheck(_window) {
+  return Boolean(_window.chrome || _window.sidebar);
+}
+
+export function isSafari(navigator) {
+  return navigator.vendor === 'Apple Computer, Inc.';
```

Comment:
This is probably OK for now, but there may be a potential bug in the future. Given that the company was renamed from `Apple Computer, Inc.` to `Apple Inc.` in 2007, it's possible that this `navigator.vendor` string may change to `Apple Inc.` in any future version of Safari, which would break this `isSafari()` function. Maybe something like `navigator.vendor.includes('Apple')` would be safer?

Reply:
Yep that's a good point will update it now

Reply:
Many thanks :white_check_mark: